vertica/tox.ini

```diff
@@ -28,3 +28,4 @@ setenv =
     9: VERTICA_VERSION=9.x
     10: VERTICA_VERSION=10.1.1-0
```

Comment:
Do you think it's worth dropping 10 if its behavior is similar to 9?

Reply:
I think we're going the way of dropping 9, rather than 10, since it's the [one on extended support](https://www.microfocus.com/productlifecycle/?term=Vertica) and it's currently the one using its own compose file and the slowest to run. But I'd rather drop it in a follow-up.

Reply:
Ah, I think we should keep testing 9 since it has different behavior in the integration